### DISCUSSION TITLE
Ability to use all elements only once

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/builder#67] [Ideas]
Then they should disappear from the element list

**Comment by cbaxter7** [comment]
Could it be introduced that specific elements are only used once also?

**Reply by adamberecz** [reply]
This would require disabling cloning functionality, [which is possible](https://builder.vueform.com/docs/custom-elements#disabling-element-features), so I've extended the example related to Discussion #66:
https://stackblitz.com/edit/github-bmamqp-72epfq?file=src%2FApp.vue

Generally if you define `schema.builder.clone = false` for an element new copies of it will become "unclonable".
